### PR TITLE
Minor addition to dev documentation

### PR DESCRIPTION
**Pull request by dgkanatsios**
Added a couple of commands to properly configure Git for a local fork + a `yarn install` run prompt for the brigade-worker directory.

### CODE REVIEW

File: docs/topics/developers.md [hunk 1]
```diff
@@ -64,7 +76,7 @@ $ make docker-build
 
 ## Javascript Bootstrap/Test
 
-To bootstrap the Javascript dependencies required by Brigade Worker:
+You should run `yarn install` in the `brigade-worker` directory. Then, to bootstrap the Javascript dependencies required by Brigade Worker:
```
[the remark this answers is not in the document]
Comment (radu-matei) [reply docs/topics/developers.md]: Yeah, I think that's a great idea.

[the remark this answers is not in the document]
Comment (dgkanatsios) [reply docs/topics/developers.md]: So, `make bootstrap-js` is not called by `make test`. However, `make test-js` is called, which calls `yarn test` which fails :(
Maybe we should have `make test-js` call `bootstrap-js`?

[the remark this answers is not in the document]
Comment (dgkanatsios) [reply docs/topics/developers.md]: If in the future we use a CI/CD pipeline (Azure DevOps/Travis) wouldn't it make sense to install all dependencies and compile the packages on `make test`? With this in mind, I'd vote for `make test-js` call `bootstrap-js` but I'll leave it to you to decide.

[the remark this answers is not in the document]
Comment (dgkanatsios) [reply docs/topics/developers.md]: - Added [O365-notify](https://hub.docker.com/r/dgkanatsios/o365-notify) to documentation
- Added `make test-js` call `bootstrap-js`

**Comment by dgkanatsios** [comment]
sorry about the force pushes, I messed EOL conversion somehow, did some squashes, it's probably OK now

**Pull request merged**